libs/community/langchain_community/chat_models/tongyi.py

```diff
@@ -88,8 +96,14 @@ def convert_dict_to_message(
                     )
         else:
             additional_kwargs = {}
+
         return (
-            AIMessageChunk(content=content)
+            AIMessageChunk(
+                content=content,
+                additional_kwargs=additional_kwargs,
+                tool_calls=tool_calls,
```

Comment:
are these tool calls fully formed in a streaming context?

if not you can specify `tool_call_chunks` on AIMessageChunk instead, with `args` a (partial json) string. See example here: https://github.com/langchain-ai/langchain/blob/c010ec8b71771dc3f54dc148475c90070b6a7c0b/libs/partners/openai/langchain_openai/chat_models/base.py#L264

Reply:
Thanks a lot, I have made some updates about this.